source/develop/release-management/features/integration/genericregistration.html.md

```diff
@@ -132,4 +132,3 @@ autoinstall: Use systemd service
 
 
 Comments and discussion can be posted on mailinglist
-
```

Comment:
This change is unrelated to this patch. Please remove it (and perform it in a different patch if it's an intended change)